README.md

```diff
@@ -1,2 +1,70 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) State what task this algorithm performs.
+
+b) Explain why the worst case running time of the algorithm is quadratic, i.e.
+O(n^2).
+it's n to the power of 2 because it has to loop through i and then loop through again for j
```

Comment:
✔️ -- so for each element of the list, it loops through the entire list again resulting in n*n operations